doc/stake-pool-operations/start_your_nodes.md

```diff
@@ -23,4 +23,6 @@ Start a __block producing__ node with:
     --shelley-vrf-key vrf.skey \
     --shelley-operational-certificate node.cert
 
-Please note that when running a node, it is important to use process monitoring so that the node can be automatically restarted when it terminates unexpectedly. We recommend that node operators run their nodes using a standard service monitoring or supervisor tool which will automatically restart the node in case of failure (such as temporary disk corruption), thereby avoiding downtime. We recommend systemd on Linux, docker’s auto-restart functionality, or any equivalent process monitoring tool. This does not apply to Daedalus users, as the node process monitoring is handled by Daedalus itself.
+Please note that when running a node, it is important to use process monitoring so that the node can be automatically restarted when it terminates unexpectedly. The node relies on being restarted as part of its mechanism for automatically recovering from disk file corruption or other unexpected conditions. In case the node process is not automatically restarted, the node will not be available to be used by stake pool operators to create blocks or exchange wallet information, for example. 
```

Comment:
```suggestion
Please note that when running a node, it is important to use process monitoring so that the node can be automatically restarted when it terminates unexpectedly. The node relies on being restarted as part of its mechanism for automatically recovering from disk file corruption or other unexpected conditions. In case the node process is not automatically restarted, the node will not be available. Thus for example, stake pools will not be able to create blocks and exchanges will not be able to get wallet information.
```